packages/pipeline-editor/src/PipelineController/index.ts

```diff
@@ -458,7 +458,7 @@ class PipelineController extends CanvasController {
         const valueIndex = schema.enum.indexOf(propValue);
         const propLabel = schema.enumNames?.[valueIndex] ?? propValue;
         if (propLabel) {
-          draft["ui:placeholder"] = `${propLabel} (pipeline default)`;
+          draft["ui:placeholder"] = `Use Pipeline Default: [${propLabel}]`;
```

Comment:
A couple nits based on previous discussion: `pipeline default` should be lower case here (since it's a phrase not a title) and there should be a space before and after `${propLabel}`

Reply:
nvm the spaces, just saw the updated screen shots and it looks really good with them